### DISCUSSION TITLE
Is there support for Federatation in Async-Graphql dynamic schemas?

### DISCUSSION THREAD

**Discussion by AdamJSoftware** [async-graphql/async-graphql#1332] [Q&A] [answered]
If so, how do you do it? If not, will it ever be supported and if so when?

**Comment by sunli829** [comment]
federation1 is supported, but version 2 is not.

**Reply by AdamJSoftware** [reply]
Is there any examples/documentation on how to implement it?

**Comment by sunli829** [comment, marked as answer]
I've added an example for Federation**2**

https://github.com/async-graphql/examples/tree/2d21325566bf07a14ca6b3727354f1ba838438ab/federation/dynamic-schema

**Reply by AdamJSoftware** [reply]
couldn't ask for anything more
